content/articles/how-to-create-an-android-wear-os-app/index.md

```diff
@@ -0,0 +1,269 @@
+### Introduction
+We normally concentrate on only two types of devices when developing mobile apps: tablets and smart phones. Have you ever considered how to build applications for wearables?
+
+Smart watches varies from smart phones in terms of design and interactivity due to distinct usage circumstances and smaller screens. The layout is simpler and more reliant on swiping actions to function. You'll learn how to make an app for a Wear OS  wearable device in this article.
+
+### Prerequisites
+The reader should:
+- Have Android studio installed
```

Comment:
Could you please paraphrase this? It would sound better as, "To follow through this tutorial, the reader should..."